recipes/physx/4.x.x/conanfile.py

```diff
@@ -98,56 +86,83 @@ def validate(self):
                 )
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version],
-                  destination=self._source_subfolder, strip_root=True)
+        get(self, **self.conan_data["sources"][self.version], destination=self.source_folder, strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+
+        tc.cache_variables["CMAKE_POSITION_INDEPENDENT_CODE"] = self.options.get_safe("fPIC", True)
```

Comment:
This should be managed by Conan? Why is it required?

Reply:
Keep it please, it's here on purpose.

Reply:
Thanks for the feedback @SpaceIm, that makes more sense now 🙂

Reply:
Lets add a comment for the next person :)

```suggestion
        # Needed. See https://github.com/conan-io/conan-center-index/pull/16583/files#r1188548265 for details
        tc.cache_variables["CMAKE_POSITION_INDEPENDENT_CODE"] = self.options.get_safe("fPIC", True)
```